cylc/flow/scripts/cylc_broadcast.py

```diff
@@ -262,7 +262,7 @@ def get_option_parser():
         "-e", "--expire", metavar="CYCLE_POINT",
         help="Cancel any broadcasts that target cycle "
              "points earlier than, but not inclusive of, CYCLE_POINT.",
-        action="store", default=None, dest="expire")
+        action="store", default='', dest="expire")
```

Comment:
@dwsutherland I had a functional test (`broadcast/00-simple.t`) failing as `bCutoff` (which is created using this `expire` value) was set to `None`.

In `cylc-flow`, in `network/graphql.py`, the `get_variable_values` function removes variables that are set to `None`. Which results in a GraphQL without all the variables defined.

If you run a query in JavaScript, or in GraphiQL and the number of variables in the query definition does not match the number of variables provided in the client query, it results in an error.

Apparently it was working for this functional test, but I think it's safe to set the default here to an empty string? The tests pass fine. WDYT?

Reply:
Probably want to change this back now?

Reply:
I've dropped the commit with that change, and push-force'd. Now it's up to GitHub actions to confirm everything's working :+1: